content/10.methods.md

```diff
@@ -102,6 +102,30 @@ This prediction represents the probability of an honoree or author selecting a c
 
 ### Estimation of Nationality
 
+We also applied the Python package `ethnicolr` to estimate race and ethnicity, as developed by Sood and Laohaprapanon. [@arxiv:1805.02109]
+`ethnicolr` also uses United States Census data, but also adds in data curated from Wikipedia in 2009. [@doi:10.1145/1557019.1557032]
+While ethnicolr overcomes some of the limitations of wru, its international representation is still limited.
+For instance, 76% of the names in ethnicolr’s Wikipedia dataset are European in origin, and the dataset contains substantially fewer Asian, African, and Middle Eastern names. 
```

Comment:
```suggestion
For instance, 76% of the names in ethnicolr’s Wikipedia dataset are European in origin, and the dataset contains remarkably fewer Asian, African, and Middle Eastern names compared to that of wru. 
```
is this what you mean? I tried to avoid using "substantially" twice in 3 sentences, but there may be a better adverb.